Project.toml

```diff
@@ -4,7 +4,9 @@ authors = ["Tim Kittel", "Jan Liße", "Sabine Auer"]
 version = "2.2.0"
 
 [deps]
+CSTParser = "00ebfdb7-1f24-5e51-bd34-a7502290713f"
```

Comment:
Why is this new dependency needed?

Reply:
I forgot to remove all the unnecessary dependencies you mentioned and made up for this now in this [commit](https://github.com/JuliaEnergy/PowerDynamics.jl/pull/55/commits/b1e67c4e0c5b4f7b8dd9c0648ba1db0344de99a8)